### DISCUSSION TITLE
--release flag

### DISCUSSION THREAD

**Discussion by ghost** [bevyengine/bevy#7004] [Ideas]
You could also add release option to the bevy fast compile guide.

```bash
cargo run --release
```

It will compile for production but has the most optimizations which is good. 

App runs faster, I just checked, you can check too

Of course it compiles it first, but second time and  after that just flying

**Comment by soqb** [comment]
we encourage users not to run with `--release` as it disables debug information which is rather important during development. instead you can follow [the instructions on the cheatbook](https://bevy-cheatbook.github.io/pitfalls/performance.html) for running with debug info and optimisations.
